rest/server_test.go

```diff
@@ -153,7 +177,10 @@ func TestSecretRestAPI_Post(t *testing.T) {
 	for _, tt := range testCases {
 		t.Run(tt.name, func(t *testing.T) {
 			log.Println(tt.name)
-			serverPort := strconv.Itoa(rand.Intn(20000-10000) + 10000)
+			serverPort, err := CheckPort()
+			if err != nil {
+				panic("sdfsdf")
```

Comment:
Instead of panic it looks more predictable to make another attempt to finally find open port. WDYT?

Reply:
> Instead of panic it looks more predictable to make another attempt to finally find open port. WDYT?

i think it is not necessary because checkFile already. I've  added require.FailNow instead